packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -224,6 +297,39 @@ export class KubernetesClient {
     }
   }
 
+  async listPods(): Promise<PodInfo[]> {
+    const ns = this.getCurrentNamespace();
+    if (ns !== undefined) {
+      const pods = await this.listNamespacedPod(this.getCurrentNamespace() as string);
```

Comment:
better with `const pods = await this.listNamespacedPod(ns);`

Reply:
agree with @lstocchi